neqo-http3/src/server_events.rs

```diff
@@ -84,6 +84,19 @@ impl StreamHandler {
             .send_data(self.stream_id(), buf, &mut self.conn.borrow_mut())
     }
 
+    /// Bytes sendable on stream at the QUIC layer.
+    ///
+    /// Note that this does not yet account for HTTP3 frame headers.
+    ///
+    /// # Errors
+    ///
+    /// It may return `InvalidStreamId` if a stream does not exist anymore.
+    pub fn available(&mut self) -> Res<usize> {
+        let stream_id = self.stream_id();
+        let n = self.conn.borrow_mut().stream_avail_send_space(stream_id)?;
+        Ok(n)
+    }
+
```

Comment:
Note that this is for testing only. Though, given that it is used in `neqo-http3/tests` and not `neqo-http3/src` it can not be prefixed with `#[cfg(tests)]`. I assume this is fine, given that all our server logic is for testing purposes only anyways. This includes this file, i.e. `neqo-http3/src/server_events.rs`.